src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -830,6 +839,20 @@ export default class SourceFileLinter {
 				!VALID_TESTSUITE.test(this.sourceFile.fileName) &&
 				symbolName === "ready" && moduleName === "sap/ui/core/Core") {
 				this.#reportTestStarter(node);
+			} else if (symbolName === "applySettings" &&
+				nodeType.symbol?.declarations?.some((declaration) =>
+					this.isUi5ClassDeclaration(declaration, "sap/ui/core/Control"))) {
```

Comment:
All related methods and functionality is implemented on the ManagedObject level, so we should always check for sub-classes of ManagedObject, not Control.

Reply:
Fixed with: https://github.com/SAP/ui5-linter/pull/513